tests/acceptance/features/bootstrap/SharingNgContext.php

```diff
@@ -92,33 +92,62 @@ public function createLinkShare(string $user, TableNode $body): ResponseInterfac
 	}
 
 	/**
-	 * @When /^user "([^"]*)" gets permissions list for (folder|file) "([^"]*)" of the space "([^"]*)" using the Graph API$/
-	 *
 	 * @param string $user
 	 * @param string $fileOrFolder   (file|folder)
 	 * @param string $resource
 	 * @param string $space
 	 *
-	 * @return void
+	 * @return ResponseInterface
 	 * @throws Exception
 	 */
-	public function theUserPermissionsListOfResource(string $user, string $fileOrFolder, string $resource, string $space):void {
+	public function getPermissionsList(string $user, string $fileOrFolder, string $resource, string $space):ResponseInterface {
```

Comment:
if `$resource` is optional then it can go to the last. Also fix the doc above
```suggestion
	public function getPermissionsList(string $user, string $fileOrFolder, string $space, string $resource):ResponseInterface {
```